### PR TITLE
Added footer and body 

### PR DESCRIPTION
**Pull request by Mike47ip**
All remaining key elements of the body and footer section of the Portfolio page was been implemented and designed to finalized the entire project. Which includes fill in sections, personal social media handle links and Multiple portfolio posts.

### CODE REVIEW

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @Mike47ip, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive Pull Request (PR). The PR descriptions should contain a summary of the changes you have in the project/PR. This gives the reviewer an idea of the changes that took place in this particular PR. Kindly fix this before requesting another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ofarouq310 — COMMENTED** [review]
Hi @Mike47ip,✋ 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## 🔴  Invalid Code Review Request

giphy

- You have submitted a project, that does not have a descriptive pull request title,  your Pull request title should be short, and concise, and addresses what you are implementing, in your case, it could be, `Finish mobile version` or something similar, simpler. Please refer to the next picture to help you how to edit the title.
image

- Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @Mike47ip >, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that is invalid because : 
- The branch name you have chosen does not provide a clear indication of the feature or action being implemented. It is important to use descriptive branch names that accurately describe the purpose of the branch. For example, instead of a generic name like "feature1," consider using names like "**created\_sidebar**" or "**added\_comments\_section**" that convey the specific feature or action being worked on. :pray: Please rename the branch to a meaningful name. Check out this [official document](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) for further help. :+1:
- You have an invalid readme file. Please update your readme file as the readme template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Mike47ip** [comment]
Come on man. I am running out of time here.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+        <div class="vector">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+      </nav>
+    </header>
+
+    <main>
```
Comment (shahadat3669) [inline index.html:25]: Greate Ui! background color should be `#fff`. please consider updating the background color as per the design.👍 
| Your | Design|
| --- | --- |
| image | image   |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+          <div class="card">
+            <div class="title">
+              <h2 class="text">Multi-Post Stories Gain+Glory</h2>
+              <ul class="server">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button" class="button">See Project</button>
```
Comment (shahadat3669) [inline index.html:93]: Greate button! it has active stage styles 👍 . But you also need to consider adding hover stage styles. please consider adding hover styles as per in the design. 👍 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+            </div>
+          </div>
```
Comment (shahadat3669) [inline index.html:95]: Well done! 👍 Greate card. But for semantic HTML it is better to use the article tag instant of div. please consider updating it. 🙏 check this [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).

File: README.md [hunk 4]
```diff
@@ -1 +1,144 @@
...
+I would like to thank all Micronauts for their support
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Kwaku28/Portfolio-Mobile/blob/toolbar-headline-section/MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```
Comment (shahadat3669) [inline README.md]: Please remove the unnecessary text. 🙏 It will be best for visitors.

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi  @Mike47ip , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 

## Highlights

✅ Descriptive PR title and summary.
✅ No linters errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [Optional] The branch name you have chosen does not provide a clear indication of the feature or action being implemented. It is important to use descriptive branch names that accurately describe the purpose of the branch. For example, instead of a generic name like "feature1," consider using names like "**created\_sidebar**" or "**added\_comments\_section**" that convey the specific feature or action being worked on. :pray: Please rename the branch to a meaningful name. Check out this [official document](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) for further help. :+1:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jmonto55 — APPROVED** [review]
# STATUS: APPROVED 🟢

Hi @Mike47ip, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

AgreeSealGIF

## Highlights

- [ ] Great UI, it looks just like the Figma design.✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by skyv26 — APPROVED** [review]
## Hi @Mike47ip, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 
- Well-structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**